File: src/main/java/no/nav/domain/pensjon/regler/beregning2011/Skjermingstillegg.java
package no.nav.domain.pensjon.regler.beregning2011;

import no.nav.domain.pensjon.regler.beregning.Ytelseskomponent;
import no.nav.domain.pensjon.regler.kode.YtelsekomponentTypeCti;

public class Skjermingstillegg extends Ytelseskomponent {



    private Double ft67Soker;

    private Double skjermingsgrad;

    private Integer ufg;

    private Double basGp_bruttoPerAr;

    private Double basTp_bruttoPerAr;

    private Double basPenT_bruttoPerAr;

    public Skjermingstillegg() {
        super();
        ytelsekomponentType = new YtelsekomponentTypeCti("SKJERMT");
    }


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/beregning2011/Skjermingstillegg.kt
package no.nav.pensjon.regler.domain.beregning2011

import no.nav.pensjon.regler.domain.beregning.Ytelseskomponent
import no.nav.pensjon.regler.domain.enum.YtelseskomponentTypeEnum

class Skjermingstillegg : Ytelseskomponent() {
    var ft67Soker: Double? = null
    var skjermingsgrad: Double? = null
    var ufg: Int? = null
    var basGp_bruttoPerAr: Double? = null
    var basTp_bruttoPerAr: Double? = null
    var basPenT_bruttoPerAr: Double? = null

    override var ytelsekomponentTypeEnum: YtelseskomponentTypeEnum = YtelseskomponentTypeEnum.SKJERMT

}